### PR TITLE
Updated with Xcode 8, Swift 2.3, iOS Target moved to iOS 8

### PR DESCRIPTION
**Pull request by madhavajay**
Needed this for Updating Quick.

### CODE REVIEW

**Comment by stephenyao** [comment]
Also waiting on this, +1 for merging through

File: Nimble.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1601,6 +1603,7 @@
 				PRODUCT_NAME = Nimble;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 2.3;
```
Comment (ikesyo) [inline Nimble.xcodeproj/project.pbxproj]: This setting is already set at the project level, so should not be set at each target level.

File: Nimble.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1517,7 +1518,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```
Comment (ikesyo) [inline Nimble.xcodeproj/project.pbxproj]: The framework intentionally support iOS 7, see #118 and #262.

File: Nimble.xcodeproj/xcshareddata/xcschemes/Nimble-macOS.xcscheme [hunk 3]
```diff
@@ -16,7 +16,7 @@
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "1F925EAC195C0D6300ED456B"
                BuildableName = "Nimble.framework"
-               BlueprintName = "Nimble-OSX"
+               BlueprintName = "Nimble-macOS"
```
Comment (ikesyo) [inline Nimble.xcodeproj/xcshareddata/xcschemes/Nimble-macOS.xcscheme]: The OS X (macOS) CI builds are broken now with the scheme name change. 😢 

- https://travis-ci.org/Quick/Nimble/jobs/149146818
- https://travis-ci.org/Quick/Nimble/jobs/149146824

**Comment by ikesyo** [comment]
Xcode 8 and Swift 2.3 are already supported on `master` by #302 and #316.

**Comment by ikesyo** [comment]
Superseded by #321. Closing :pray:

**Pull request closed without merging**